frontend/src/components/common/HeaderLabel.tsx

```diff
@@ -0,0 +1,25 @@
+import { Box, Text } from "@chakra-ui/react";
+import React from "react";
+
+interface HeaderLabelProps {
+  text: string;
+  isDesktop: boolean;
+}
+const HeaderLabel = ({ text, isDesktop }: HeaderLabelProps) => {
+  if (isDesktop) {
+    return (
+      <Box textAlign="center">
+        <Text mt="67px" textStyle="mobileHeader1">
+          {text}
+        </Text>
+      </Box>
+    );
+  }
+  return (
+    <Text mt="67px" textStyle="mobileHeader1">
+      {text}
+    </Text>
+  );
+};
```

Comment:
seems like you're using `HeaderLabel` in a few places so it's fine to extract like this 
but I don't think we need to pass unnecessary props for it, can do something like:
let me know if it doesn't work?
```suggestion
const HeaderLabel = ({ text }: HeaderLabelProps) =>(
    <Text mt="67px" textStyle="mobileHeader1" textAlign={["inherit", "center"]}>
      {text}
    </Text>
  );
```